src/dlsproto/client/request/internal/GetRange.d

```diff
@@ -169,13 +169,20 @@ public struct GetRange
     {
         auto context = This.getContext(context_blob);
 
+        Notification notification;
+
         if (!context.shared_working.stopped)
         {
             // Final notification, after the request has been finished
-            Notification notification;
             notification.finished = NoInfo();
-            GetRange.notify(context.user_params, notification);
         }
+        else
+        {
+            // Stopped notification instead of the final one
+            notification.stopped = NoInfo();
+        }
```

Comment:
I'd suggest to change the if-statement to the code below instead (if the current changes are approved):
```D
        if (context.shared_working.stopped)
        {
            // Stopped notification instead of the final one
            notification.stopped = NoInfo();
        }
        else
        {
            // Final notification, after the request has been finished
            notification.finished = NoInfo();
        }
```

Reply:
Updated, thanks!